Makefile

```diff
@@ -10,7 +10,7 @@ linux:
 
 image: linux
 	cp servicebroker image/
-	docker build image/ -t quay.io/osb-starter-pack/servicebroker
+	sudo docker build image/ -t quay.io/osb-starter-pack/servicebroker
```

Comment:
I would really prefer not to merge the sudo command; i’d be happy to have a link to how to add your user to the docker group, though

Reply:
@pmorie I thought the official recommendation from Red Hat is to not add your user to the docker group.

https://www.projectatomic.io/blog/2015/08/why-we-dont-let-non-root-users-run-docker-in-centos-fedora-or-rhel/